### PR TITLE
FAQ's added

### LINKED ISSUE #55: Add FAQ section to BUB
Add faq section to README.md . This would be an open issue for some time where new FAQs would be added to repo. After some FAQs(max 5), redirect to a FAQ.md file in the repo.

### PR DESCRIPTION
**Pull request by habeeba-naaz**
Fixes #55. Please review and let me know any changes to be done.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<b>Frequently Asked Questions</b>
+<b>What is Book Uploader Bot?<b>
+A Book Uploader Bot transfers documents from public libraries such as Google Books, and Punjab Digital Library etc to Internet Archive.
+<b>What is Wikimedia Tool Labs?<b> -Please answer
+<b>What are the Prerequisites?<b>
+NodeJs,Docker ToolBox,Redis
```
Comment (coderwassananmol) [inline FAQ.md]: What is docker toolbox?

Comment (coderwassananmol) [reply FAQ.md]: Pre-requisites for installation are not required in the FAQ.

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<b>Frequently Asked Questions</b>
```
Comment (coderwassananmol) [inline FAQ.md]: Just answer these questions about the tool:
1. What is Book Uploader Bot (BUB2)?
2. What does this tool do?
3. Who can benefit from this?
4. Future enhancements?

Comment (habeeba-naaz) [reply FAQ.md]: Sure.Please answer what are future enhancements?

Comment (habeeba-naaz) [reply FAQ.md]: I have answered above all questions except Future Enhancements question is left.If that is answered i'll send a final commit.

Comment (coderwassananmol) [reply FAQ.md]: > I have answered above all questions except Future Enhancements question is left.If that is answered i'll send a final commit.

Future enhancements:
Bulk upload future, MediaWiki OAuth integration, Direct uploading to Commons

**Comment by habeeba-naaz** [comment]
@Parikshit-Hooda told to combine questions discussed in issue. So i added extra FAQ which is answered in issue also

File: FAQ.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<b>What does this tool do?</b>
+The tool is built to help the community with free books that are available in the online public libraries. It makes the integration of books easier.
+<b>Who can benefit from this tool?</b>
+The tool, under the hood, compiles all the book images/pdf and its meta data at one place where it can be accessed by anyone in the world.
+<b>What are the future enhancements?</b> 
+Bulk upload future, MediaWiki OAuth integration, Direct uploading to Commons
+<b>Can I upload my own book?</b>
+If an appropriate license is attached to the book which allows it to be archived(which doesn't allow copyright infringement suits, etc.), then book can be uploaded.
+<b>Can I delete or undo an upload?</b>
+Archives are supposed to be read-only, and deleting is a 'write' operation.So it doesnot support.
```
Comment (coderwassananmol) [inline FAQ.md]: Add a space after period (.) and a space between does and not.

**Comment by Parikshit-Hooda** [comment]
@coderwassananmol please approve. PR lgtm.

**Comment by Parikshit-Hooda** [comment]
@habeeba-naaz please link this FAQ file in the main website's help section 'faq' dropdown as well. To do this, go to <project_root>/components/Header.js  (around line 78). The FAQs look good to me now. Just link the faq.md there and raise the PR. will merge.

**Comment by habeeba-naaz** [comment]
lgtm means?

**Comment by Parikshit-Hooda** [comment]
> > lgtm means?
> 
> Let's get this merged

looks good to me

**Pull request merged**